pom.xml

```diff
@@ -25,11 +25,12 @@
         <api-helper-java.version>1.1.0-rc1</api-helper-java.version>
         <rest-service-common-library.version>0.0.3</rest-service-common-library.version>
         <environment-reader-library.version>1.3.2</environment-reader-library.version>
+        <jackson.version>2.15.3</jackson.version>
```

Comment:
in the dependency report I still see `jackson-databind-2.13.15` as a Medium vulnerability
<img width="1524" alt="Screenshot 2023-11-06 at 10 56 08 am" src="https://github.com/companieshouse/efs-submission-api/assets/2736331/a7056d94-fbe7-4e21-8435-218dd05ecb16">
it's coming from `aws-java-sdk-core:jar:1.12.343`